.github/CODEOWNERS

```diff
@@ -6,24 +6,30 @@
 ###########
 
 # Catch all for non-code project files and unowned files | folders
-*      @ahsonkhan @rickwinter @joshfree
-/sdk/  @ahsonkhan @rickwinter @joshfree
+*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989
+/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989
 
 # Samples
-/samples/      @ahsonkhan @rickwinter
-/sdk/samples/  @ahsonkhan @rickwinter
+/samples/                @rickwinter @ahsonkhan @antkmsft @vhvb1989
 
-# Azure::Core
-/sdk/core/     @ahsonkhan @antkmsft @rickwinter @vhvb1989
+# PRLabel: %Azure.Core
+/sdk/core/               @rickwinter @ahsonkhan @antkmsft @vhvb1989
```

Comment:
I thought we were sorting alphabetically :)

Reply:
I'm updating these to be primary contacts.  As the service list grows individuals we won't scale if we are all tagged on everything by default.  Instead we have primary point of contact.  For cases where the service team owns the package they are listed first.  For services where we own the library the primary contact is listed first.
This ordering is similar across the SDKs.